build/dependencies.props

```diff
@@ -175,6 +175,7 @@
     <SystemThreadingTasksDataflowPackageVersion>4.9.0-preview1-26112-01</SystemThreadingTasksDataflowPackageVersion>
     <SystemThreadingTasksExtensionsPackageVersion>4.5.0-preview1-26112-01</SystemThreadingTasksExtensionsPackageVersion>
     <SystemValueTuplePackageVersion>4.5.0-preview1-26112-01</SystemValueTuplePackageVersion>
+    <SystemNetWebSocketsWebSocketProtocolPackageVersion>4.5.0-preview1-26119-06</SystemNetWebSocketsWebSocketProtocolPackageVersion>
```

Comment:
Any reason you chose this version? To keep it aligned with other corefx packages, would 4.5.0-preview1-26112-01 work?

Reply:
There is no 4.5.0-preview1-26112-01 of this package :). It was introduced in 26119

Reply:
That's what my comment about the version number above is about. Would it be best to wait for 26119 to be mirrored? Will this file be auto-updated by that? Happy to wait on that and rebase, just not clear on the best approach.

Reply:
Or you can just update other corefx packages too ;)